extensions/amp-accordion/1.0/accordion.js

```diff
@@ -35,13 +37,25 @@ const EMPTY_EXPANDED_MAP = {};
 
 const generateSectionId = sequentialIdGenerator();
 
+const CHILD_STYLE = {
```

Comment:
Is there a reason this is not in jss?

Reply:
Otherwise, should we put this in `animations.js` instead of adding it regardless if the accordion has `animate`?

Reply:
These help animations, but also stand on their own just for style. I didn't use JSS here b/c

1. These styles are very simple and assume that never need override. So, for simple things like this (just a couple of very standard CSS props) I think `style` is still very good path.
2. We only for now support JSS for shadow DOM. We could try it for light DOM as well. But this component is actually a "disconnected" component.

Reply:
Ah yes, thanks for going over this :)